common/src/api/external/mod.rs

```diff
@@ -1415,11 +1415,19 @@ pub struct VpcFirewallRule {
     pub vpc_id: Uuid,
 }
 
+/**
+ * Collection of a [`Vpc`]'s firewall rules
+ */
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct VpcFirewallRules {
+    pub rules: Vec<VpcFirewallRule>,
+}
```

Comment:
Maybe this should be called a `VpcFirewall` and the endpoint should simply be `/firewall`. That way a `Firewall` is essentially the name for a collection of rules. I don't see why we would want to have multiple firewalls per VPC, but we don't have to support that. It could just be `/vpc/:vpcName/firewall` and there is exactly one firewall.